src/_posts/languages/php/2000-01-01-extensions.md

```diff
@@ -0,0 +1,145 @@
+* Socket management: [`sockets`](https://www.php.net/manual/en/book.sockets.php)
+* Math functions: [`bcmath`](https://www.php.net/manual/book.bc.php)
+* Images manipulation: [`gd`](https://www.php.net/manual/book.image.php)
+* Intl (Internationalization): [`intl`](https://www.php.net/manual/book.intl.php)
+
+## Working With PECL Extensions
+
+All [PECL extensions](https://pecl.php.net/) are available. If the extension
+you need is not in the above list of pre-installed extensions, it will be
+compiled during the *build* phase of your deployment.
+
+To enable a native PHP extension, add it in the `require` block of your
+`composer.json`, like so (notice the `ext-` prefix):
+
+```json
+{
+  "require": {
+    "ext-mongodb": "*",
+    "ext-imagick": "*",
+    ...
+  }
+}
+```
+
+{% note %}
+- Some PECL extensions are very old, probably even unmaintained and will only
+  work with an old version of PHP (e.g. `mongo`, `sodium`, `ds` or `lua`). We
+  strongly advise you to pick your extension(s) carefully.
+- If the compilation of the extension succeeds, the compiled extension (the
+  `.so` file) is kept in cache for future deployments.
+{% endnote %}
+
+### Working With PECL Extensions That Have Dependencies
+
+While some PECL extensions compile and install just fine out of the box, some
+others have precise requirements. In most cases, these requirements consist in
+libraries and libraries headers, which are generally available and distributed
+as OS packages.
+
+The documentation of the extension should normally specify the name(s) (and
+sometimes the versions) of these required package(s).
+
+To install these packages in your container, and when possible, we **strongly**
+recommend to use the
+[APT buildpack]({% post_url platform/deployment/buildpacks/2000-01-01-apt %})
+along with the
+[multi-buildpack]({% post_url platform/deployment/buildpacks/2000-01-01-multi %}).
+
+### Specifying Compilation Flags
+
+Some extensions also require specific compilation flags to be set. To specify
+these flags, [create a new environment variable]({% post_url platform/app/2000-01-01-environment %})
+named after the extension and prefixed with
+`PHP_PECL_EXTENSION_CONFIGURE_ARGS_` (see [Full Example](#full-example), below).
+Then, put the compilation flags you want to use in this variable.
+
+Each extension requiring specific compilation flags must have its own
+environment variable.
+
+### Full Example
+
+In this example, we are going to install the PECL extension named `yaml`. This
+is a good example as it requires the`libyaml` and `libyaml-dev` packages to be
+installed, and it also needs some specific compilation flags.
+
+First, let's tell the platform we need this extension. To do so, we will have
+to create a file named `composer.json` at the root of our project, like this:
+
+```json
+{
+  "require": {
+    "ext-yaml": "*"
+  }
+}
+```
+
+Don't forget to generate the `composer.lock` file, using Composer:
+
+```bash
+composer install
+```
```

Comment:
Related to previous review: https://github.com/Scalingo/documentation/pull/2139#discussion_r1217750404

```suggestion
```